movieapp/src/components/sorting/index.tsx

```diff
@@ -0,0 +1,39 @@
+type SortElement = {
+  [id: string]: string;
+  title: string;
+  link: string;
+};
+
+const sortArray: Array<SortElement> = [
```

Comment:
I'd call it as MovieSortCritarias 

use plural for lists structures

Reply:
fixed